compiler/src/main/kotlin/io/spine/protodata/ContextExtension.kt

```diff
@@ -24,14 +24,17 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata.cli.given
+package io.spine.protodata
 
-import io.spine.protodata.TypeEntered
-import io.spine.protodata.subscriber.CodeEnhancement
-import io.spine.protodata.subscriber.Subscriber
+import io.spine.server.BoundedContextBuilder
 
-class TestSubscriber : Subscriber<TypeEntered>(TypeEntered::class.java) {
+/**
+ * An extension above the `ProtoData` bounded context.
+ */
+public interface ContextExtension {
 
-    override fun process(event: TypeEntered): Iterable<CodeEnhancement> =
-        listOf(AddUnderscore(event.type.name))
+    /**
+     * Registers the extension entities in the bounded context.
+     */
+    public fun fillIn(context: BoundedContextBuilder)
```

Comment:
Do we really need the `public` modifier here and above?

Reply:
> fillIn()

Can we do it another way around, i.e. expose things that need to be registered?

Also, can I remove something from the passed builder?